src/client/Microsoft.Identity.Client/TokenResponseHelper.cs

```diff
@@ -27,7 +28,7 @@ public static string GetTenantId(IdToken idToken, AuthenticationRequestParameter
                 idToken?.TenantId).TenantId;
         }
 
-        public static string GetPreferredUsernameFromIdToken(bool isAdfsAuthority, IdToken idToken)
+        public static string GetPreferredUsernameFromIdToken(IdToken idToken)
```

Comment:
nit: rename to GetUsernameFromIdToken? Since PreferredUsername is an overloaded term.